src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx

```diff
@@ -22,26 +21,29 @@ import { useSaveArtwork } from "Components/Artwork/SaveButton/useSaveArtwork"
 import { FC, useCallback } from "react"
 import { RouterLink } from "System/Router/RouterLink"
 import { useRouter } from "System/Router/useRouter"
+import { FullscreenBox } from "Components/FullscreenBox"
 
 interface ArtQuizArtworksProps {
   me: ArtQuizArtworks_me$data
 }
 
 export const ArtQuizArtworks: FC<ArtQuizArtworksProps> = ({ me }) => {
   const { router } = useRouter()
-  const resume = me.quiz!.quizArtworkConnection!.edges!.findIndex(edge => {
+
+  const edges = me.quiz.quizArtworkConnection!.edges!
```

Comment:
I *reeeeeeaaaally* want to fix these connection types to be non-nullable. I guess we could also have a helper type to cast them that way as a temporary hack...